.buildkite/comparison/pipeline.yml

```diff
@@ -0,0 +1,291 @@
+      command:
+        - echo "$(bash .buildkite/comparison/comparison_sphere_held_suarez.sh 24 high disable Float64)"
+      artifact_paths:
+        - "comparison_sphere_held_suarez_high_res_rhoe_24/scaling_data_24_processes.jld2"
+        - "comparison_sphere_held_suarez_high_res_rhoe_24/report.24.nsys-rep"
+      env:
+        CLIMACORE_DISTRIBUTED: "MPI"
+        NPROCS: 24
+      agents:
+        config: cpu
+        queue: central
+        slurm_nodes: 1
+        slurm_ntasks: 24
+        slurm_mem: 64GB
+        slurm_exclusive:
+
+    - label: ":computer: MPI Held-Suarez high resolution test(ρe_tot) - (54) process"
+      command:
+        - echo "$(bash .buildkite/comparison/comparison_sphere_held_suarez.sh 54 high disable Float64)"
+      artifact_paths:
+        - "comparison_sphere_held_suarez_high_res_rhoe_54/scaling_data_54_processes.jld2"
+        - "comparison_sphere_held_suarez_high_res_rhoe_54/report.54.nsys-rep"
+      env:
+        CLIMACORE_DISTRIBUTED: "MPI"
+        NPROCS: 54
+      agents:
+        config: cpu
+        queue: central
+        slurm_nodes: 2
+        slurm_ntasks: 54
+        slurm_mem: 64GB
+        slurm_exclusive:
+
+    - label: ":computer: MPI Held-Suarez high resolution test(ρe_tot) - (96) process"
+      command:
+        - echo "$(bash .buildkite/comparison/comparison_sphere_held_suarez.sh 96 high disable Float64)"
+      artifact_paths:
+        - "comparison_sphere_held_suarez_high_res_rhoe_96/scaling_data_96_processes.jld2"
+        - "comparison_sphere_held_suarez_high_res_rhoe_96/report.96.nsys-rep"
+      env:
+        CLIMACORE_DISTRIBUTED: "MPI"
+        NPROCS: 96
+      agents:
+        config: cpu
+        queue: central
+        slurm_nodes: 3
+        slurm_ntasks: 96
+        slurm_mem: 64GB
+        slurm_exclusive:
+
+    - label: ":computer: MPI Held-Suarez high resolution test(ρe_tot) - (216) process"
+      command:
+        - echo "$(bash .buildkite/comparison/comparison_sphere_held_suarez.sh 216 high disable Float64)"
+      artifact_paths:
+        - "comparison_sphere_held_suarez_high_res_rhoe_216/scaling_data_216_processes.jld2"
+        - "comparison_sphere_held_suarez_high_res_rhoe_216/report.216.nsys-rep"
+      env:
+        CLIMACORE_DISTRIBUTED: "MPI"
+        NPROCS: 216
+      agents:
+        config: cpu
+        queue: central
+        slurm_nodes: 7
+        slurm_ntasks: 216
+        slurm_mem: 64GB
+        slurm_exclusive:
+
+  - wait
+
+  - label: ":computer: comparison plots (high-resolution simulations)"
+    key: "cpu_comparison_plots_high-resolution"
+    command:
+      - "julia --color=yes --project=examples examples/hybrid/comparison_plots.jl --job_id comparison_sphere_held_suarez_high_res_rhoe"
+    artifact_paths:
+      - "high-resolution_comparison.*"
+      - "high-resolution_sypd.*"
+      - "high-resolution_Scaling.*"
+      - "high-resolution_Scaling_efficiency.*"
```

Comment:
Will these need prepended `comparison_sphere_held_suarez_high_res_rhoe/`? (I'm not sure)

Reply:
Not in the CI. But it is helpful when generating plots on local computer.